configs/solo/README.md

```diff
@@ -0,0 +1,34 @@
+# SOLO: Segmenting Objects by Locations
+
+## Introduction
+
+```
+@inproceedings{wang2020solo,
+  title     =  {{SOLO}: Segmenting Objects by Locations},
+  author    =  {Wang, Xinlong and Kong, Tao and Shen, Chunhua and Jiang, Yuning and Li, Lei},
+  booktitle =  {Proc. Eur. Conf. Computer Vision (ECCV)},
+  year      =  {2020}
+}
+```
+
+## Results and Models
+
+### SOLO
```

Comment:
In order to facilitate understanding, it is recommended to add the difference between SOLO/Decoupled SOLO/Decoupled Light SOLO at the end of the document.